bahmni-docker/docker-compose.yml

```diff
@@ -137,14 +141,11 @@ services:
       - openmrsdb
 
   openmrsdb:
-    image: bahmni/openmrs-db:${OPENMRS_DB_IMAGE_TAG:?}
+    image: mysql:5.6
```

Comment:
Are we not going with mysql:5.7 here?

Reply:
When we run with mysql:5.7 on docker, it has some issues and queries are not passing, so we will keep it with 5.6 and once OMRS Upgrade is done we can move to latest mysql version

Reply:
Okay.